### DISCUSSION TITLE
When logged in Foxglove, https//foxglove.dev redirects to https://console.foxglove.dev

### DISCUSSION THREAD

**Discussion by christian-nils** [foxglove/community#935] [General]
Hello,

Thanks for all your tools made available to the community.

I have a question related to your website. When I log myself in the Foxglove Studio app, every time I want to visit your website (https://foxglove.dev), I am automatically redirected to https://console.foxglove.dev. This is quite frustrating because I would really like to be able to browse your website and not https://console.foxglove.dev.

Is this an intended behaviour? Can you please let me know how I can browse your website without having to log out or use my browser in incognito mode.

Thanks in advance,

Have a great day,

Greetings,

Christian-Nils

**Comment by amacneil** [comment]
It's intended behavior. For example, when you visit https://github.com you are shown the application, not the marketing website.

Out of interest, which pages are you trying to visit?

**Comment by christian-nils** [comment]
I was in a call with some of my colleagues and I wanted to show them the website and navigate them through the website.

Basically, I wanted to show them what Foxglove Studio and MCAP were, how to access the docs  and how to download Foxglove Studio. I could not find those links from console.foxglove.dev.

**Reply by amacneil** [reply]
Thanks for the feedback - we will look at adding links back to these pages from within the app.

In the meantime, you can always view these pages by visiting the URLs directly, e.g. https://foxglove.dev/blog, or using an incognito window.

**Reply by christian-nils** [reply]
Thanks, have a great week!
